pkg/injector/annotations/annotations.go

```diff
@@ -0,0 +1,92 @@
+	KeyCPURequest                   = "dapr.io/sidecar-cpu-request"
+	KeyMemoryRequest                = "dapr.io/sidecar-memory-request"
+	KeySidecarListenAddresses       = "dapr.io/sidecar-listen-addresses"
+	KeyLivenessProbeDelaySeconds    = "dapr.io/sidecar-liveness-probe-delay-seconds"
+	KeyLivenessProbeTimeoutSeconds  = "dapr.io/sidecar-liveness-probe-timeout-seconds"
+	KeyLivenessProbePeriodSeconds   = "dapr.io/sidecar-liveness-probe-period-seconds"
+	KeyLivenessProbeThreshold       = "dapr.io/sidecar-liveness-probe-threshold"
+	KeyReadinessProbeDelaySeconds   = "dapr.io/sidecar-readiness-probe-delay-seconds"
+	KeyReadinessProbeTimeoutSeconds = "dapr.io/sidecar-readiness-probe-timeout-seconds"
+	KeyReadinessProbePeriodSeconds  = "dapr.io/sidecar-readiness-probe-period-seconds"
+	KeyReadinessProbeThreshold      = "dapr.io/sidecar-readiness-probe-threshold"
+	KeySidecarImage                 = "dapr.io/sidecar-image"
+	KeyAppSSL                       = "dapr.io/app-ssl"
+	KeyHTTPMaxRequestBodySize       = "dapr.io/http-max-request-size"
+	KeyHTTPReadBufferSize           = "dapr.io/http-read-buffer-size"
+	KeyGracefulShutdownSeconds      = "dapr.io/graceful-shutdown-seconds"
+	KeyEnableAPILogging             = "dapr.io/enable-api-logging"
+	KeyUnixDomainSocketPath         = "dapr.io/unix-domain-socket-path"
+	KeyVolumeMountsReadOnly         = "dapr.io/volume-mounts"
+	KeyVolumeMountsReadWrite        = "dapr.io/volume-mounts-rw"
+	KeyDisableBuiltinK8sSecretStore = "dapr.io/disable-builtin-k8s-secret-store" //nolint:gosec
+	KeyEnableAppHealthCheck         = "dapr.io/enable-app-health-check"
+	KeyAppHealthCheckPath           = "dapr.io/app-health-check-path"
+	KeyAppHealthProbeInterval       = "dapr.io/app-health-probe-interval"
+	KeyAppHealthProbeTimeout        = "dapr.io/app-health-probe-timeout"
+	KeyAppHealthThreshold           = "dapr.io/app-health-threshold"
+	KeyPlacementHostAddresses       = "dapr.io/placement-host-address"
+)
+
+// Default values
+
+const (
+	DefaultLogLevel                     = "info"
+	DefaultLogAsJSON                    = false
+	DefaultAppSSL                       = false
+	DefaultAppProtocol                  = "http"
+	DefaultEnableMetric                 = true
+	DefaultMetricsPort                  = 9090
+	DefaultEnableDebug                  = false
+	DefaultDebugPort                    = 40000
+	DefaultSidecarListenAddresses       = "[::1],127.0.0.1"
+	DefaultHealthzProbeDelaySeconds     = 3
+	DefaultHealthzProbeTimeoutSeconds   = 3
+	DefaultHealthzProbePeriodSeconds    = 6
+	DefaultHealthzProbeThreshold        = 3
+	DefaultMtlsEnabled                  = true
+	DefaultEnableAPILogging             = false
+	DefaultEnableProfiling              = false
+	DefaultDisableBuiltinK8sSecretStore = false
+	DefaultEnableAppHealthCheck         = false
+	DefaultAppCheckPath                 = "/health"
+	DefaultAppHealthProbeInterval       = 5   // in seconds
+	DefaultAppHealthProbeTimeout        = 500 // in ms
```

Comment:
Shall we update these?
```suggestion
	DefaultAppHealthProbeIntervalSeconds            = 5
	DefaultAppHealthProbeTimeoutMilliSeconds        = 500
```

Reply:
I agree.

Reply:
I will change that

Reply:
Done